environment/src/main/java/com/sequenceiq/environment/environment/repository/AzureDiskEncryptionParametersRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.sequenceiq.environment.environment.repository;
+
+import javax.transaction.Transactional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.sequenceiq.cloudbreak.workspace.repository.EntityType;
+import com.sequenceiq.environment.parameters.dao.domain.AzureParameters;
+
+@EntityType(entityClass = AzureParameters.class)
+@Transactional(Transactional.TxType.REQUIRED)
+public interface AzureDiskEncryptionParametersRepository extends JpaRepository<AzureParameters, Long> {
+}
```

Comment:
What is the reason for introducing this new repository? The usual approach is to treat the `Environment` entity as the single point of entry; nested child entities are not manipulated individually. Furthermore, the class name is also confusing; `AzureParameters` contains other things not related to disk encryption.

Reply:
we already have AzureParametersRepository anyway

Reply:
I have removed new repository, using AzureParametersRepository now. It is required because environmentService.save(environment) is not saving the encryptionKeyUrl as it is a secret. In order to set azureParameters.encryptionKeyUrl, AzureParametersRepository is used.